test/server/cli-argument-parser-test.js

```diff
@@ -289,7 +289,7 @@ describe('CLI argument parser', function () {
     });
 
     it('Should parse command line arguments', function () {
-        return parse('-r list -S -q -e --hostname myhost --qr-code --app run-app --speed 0.5 ie test/server/data/file-list/file-1.js')
+        return parse('-r list -S -q -e --hostname myhost --qr-code --app run-app --speed 0.5 ie test/server/data/file-list/file-1.js --proxy-host localhost:2000')
```

Comment:
Should we add server tests for `use Proxy` in `test\server\runner-test.js`? What do you think?